src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -141,6 +142,11 @@ void
 ServerHandlerImp::onStop()
 {
     m_server->close();
+    {
+        std::unique_lock<std::mutex> lock(mutex_);
+        condition_.wait(lock, [this]() { return stopped_; });
```

Comment:
Nit: I'll suggest using similar naming across classes on variables relating to stopping. ```cv_, wakeup_, condition_``` could be changed to ```_condition```.  ```isTimeToStop, stop_, stopping_``` could be changed to ```stop_``` etc...

Reply:
I want that too, eventually, but I want to do it separately so that the two changesets are easy to review: every change in this one is meaningful, there is no clutter of noisy renames, while every change in that one will be just a rename and easy to skim through.